kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -743,14 +729,6 @@ object KyuubiConf {
       .toSequence()
       .createWithDefault(Nil)
 
-  val FRONTEND_THRIFT_HTTP_ALLOW_USER_SUBSTITUTION: ConfigEntry[Boolean] =
```

Comment:
@mahmoudbahaa This configuration was introduced in #2815, could you please help us see if it is used?